sdk/eventgrid/eventgrid-system-events/review/eventgrid-system-events.api.md

```diff
@@ -108,15 +108,15 @@ export interface AcsChatThreadCreatedEventData extends AcsChatThreadEventInThrea
     createdByCommunicationIdentifier: CommunicationIdentifierModel;
     metadata: Record<string, string>;
     participants: AcsChatThreadParticipant[];
-    properties: Record<string, unknown>;
+    properties: Record<string, any>;
```

Comment:
👋 why are we moving from `unknown` to `any` here and everywhere else?

Reply:
@joheredi I think this is an emitter change to avoid painful casts everywhere is that right?

Reply:
Yes, we are doing any for output to ease the burden on customers. Otherwise they have to cast every output

Reply:
The problem I have with `any` is how it spreads through a codebase like wildfire 😄 

As a developer using an SDK I would personally prefer to deal with the unknowns right then and there at the integration point instead of leaking `any` throughout my codebase (since anything you'll assign this to will become `any`, and the things interacting with it become `any` etc.) or using the wrong type (which TS will happily "cast" it to) 

I trust y'alls judgement that the tradeoffs are worth it but felt obligated to bring this up at least once 🤝

Reply:
Will take this offline about unknown vs any.